webapp/packages/plugin-navigation-tree/src/NavigationTree/ElementsTree/ElementsTreeTools/ElementsTreeTools.tsx

```diff
@@ -7,63 +7,41 @@
  */
 import { observer } from 'mobx-react-lite';
 import React, { useState } from 'react';
-import styled, { css, use } from 'reshadow';
+import styled from 'reshadow';
 
-import { ACTION_ICON_BUTTON_STYLES, IconButton, PlaceholderElement, useResource, useStyles, useTranslate } from '@cloudbeaver/core-blocks';
+import {
+  ACTION_ICON_BUTTON_STYLES,
+  IconButton,
+  IconButtonStyles,
+  PlaceholderElement,
+  s,
+  SContext,
+  StyleRegistry,
+  useS,
+  useStyles,
+  useTranslate,
+} from '@cloudbeaver/core-blocks';
 import type { ComponentStyle } from '@cloudbeaver/core-theming';
 import { useCaptureViewContext } from '@cloudbeaver/core-view';
 
 import { DATA_CONTEXT_ELEMENTS_TREE } from '../DATA_CONTEXT_ELEMENTS_TREE';
 import type { IElementsTree } from '../useElementsTree';
 import { ElementsTreeFilter } from './ElementsTreeFilter';
+import ElementsTreeToolsStyles from './ElementsTreeTools.m.css';
 import { ElementsTreeToolsMenu } from './ElementsTreeToolsMenu';
 import { DATA_CONTEXT_NAV_TREE_ROOT } from './NavigationTreeSettings/DATA_CONTEXT_NAV_TREE_ROOT';
 import type { IElementsTreeSettingsProps } from './NavigationTreeSettings/ElementsTreeSettingsService';
 import { NavigationTreeSettings } from './NavigationTreeSettings/NavigationTreeSettings';
 
-const toolsStyles = css`
-  [|primary] {
-    composes: theme-text-primary from global;
-  }
-  tools {
-    composes: theme-background-surface from global;
-    display: block;
-    position: sticky;
-    top: 0;
-    z-index: 1;
-  }
-  actions {
-    display: flex;
-    flex-direction: row;
-  }
-  fill {
-    flex: 1;
-  }
-  IconButton {
-    & Icon,
-    & StaticImage {
-      transition: transform 0.3s ease-in-out;
-    }
-
-    &[|opened] Icon,
-    &[|opened] StaticImage {
-      transform: rotate(180deg);
-    }
-
-    &[|loading] Icon,
-    &[|loading] StaticImage {
-      animation: rotating 1.5s linear infinite;
-    }
-  }
-  @keyframes rotating {
-    from {
-      transform: rotate(0deg);
-    }
-    to {
-      transform: rotate(360deg);
-    }
-  }
-`;
+const registry: StyleRegistry = [
+  [
+    IconButtonStyles,
+    {
+      mode: 'append',
+      styles: [ElementsTreeToolsStyles],
```

Comment:
please split into two style files `ElementsTreeTools.m.css` and `ElementsTreeToolsIconButton.m.css`